doc/man/task-sync.5.in

```diff
@@ -143,7 +143,9 @@ Then configure Taskwarrior with:
 
 To synchronize your tasks to AWS, select a region near you and use the AWS
 console to create a new S3 bucket. The default settings for the bucket are
-adequate.
+adequate, however, ensure that no lifecycle policies are enabled, as they
```

Comment:
```suggestion
adequate. In particular, ensure that no lifecycle policies are enabled, as they
```
Just fixing a comma splice. Also the original reads a little bit like "defaults are ok except lifecycle" which might result in people new to AWS reading a lot more docs than necessary